src/goto-cc/gcc_cmdline.cpp

```diff
@@ -49,6 +49,7 @@ const char *goto_cc_options_without_argument[]=
   "--big-endian",
   "--no-arch",
   "--partial-inlining",
+  "--validate-goto-model",
```

Comment:
Please add that to all modes, not just the GCC mode.

Reply:
done